plugins/adobe-analytics-response-events/index.ts

```diff
@@ -67,16 +75,21 @@ import { ValidationPluginResult } from '../../types/validationPlugin';
         eventData['global.privacy'] ||
         event.payload.metadata?.['state.data']?.['global.privacy'];
       return privacy === 'optedin';
-    });
+    });    
 
     if (!optedin) {
       notMatchedMessage +=
-        ' If your report suite is not timestamp enabled, hits are discarded until the privacy status changes to `optedin`';
+        ' If your report suite is not timestamp enabled, hits are discarded until the privacy status changes to `optedin`.';
       links.push({
         type: 'doc',
         url: 'https://developer.adobe.com/client-sdks/documentation/adobe-analytics/faq/#verify-current-privacy-status'
       });
     }
+
+    if (!versionEvents.some(aepMobile.sharedStateVersions.getExtensionsKey('"com.adobe.module.analytics"'))) {
+      notMatchedMessage +=
+        ' If you are not using the Analytics Extension or if you follow an Edge Bridge workflow, you can disregard this validation error.';
+    }    
```

Comment:
Let's use `unknown` rather than `not matched` since it's unknown if the customer is using bridge or analytics.